package.json

```
@@ -38,7 +38,7 @@
     "stylelint-config-recommended": "^2.2.0",
     "stylelint-scss": "^3.6.1",
     "stylelint-webpack-plugin": "^0.10.5",
-    "svg-spritemap-webpack-plugin": "^3.3.1",
+    "svg-spritemap-webpack-plugin": "<=3.3.5",
```

まあ細い話なんですが、 3.3.x のパッチ更新された場合に、
こうしておけば、それを追従できる...？（よくわかってない

```suggestion
    "svg-spritemap-webpack-plugin": "~3.3",
```

ref: https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1

---

どこのバージョン以降でバグが発生するか、実はちゃんとは確認できていないので、
いったん確実に動くバージョン以下、としています。
なるはやでIssue出してきますのでお待ちを。。

---

（すでに3.3.6以上のパッチもリリースされているです）

---

現状だと class が子要素についてるけど、`display: none;` されるだけなので別にこのままでもいいのでは？感もありますね 🤔  ちょっと気持ち悪いけど

> 大元spriteを隠すためのclassが、sprite内のSVGにまで付加されてしまう。

---

子要素にdisplay: noneがつくことで、useする側まで表示されなくなることが、解決したい問題なんです。

---

子要素にdisplay: noneがつくことで、useする側まで表示されなくなることが、解決したい問題なんです。